src/main/java/eu/dissco/core/digitalmediaobjectprocessor/configuration/ApplicationConfiguration.java

```diff
@@ -24,7 +26,16 @@ public Random random() {
   @Bean
   public DocumentBuilder documentBuilder() throws ParserConfigurationException {
     var docFactory = DocumentBuilderFactory.newInstance();
+    docFactory.setFeature("http://apache.org/xml/features/disallow-doctype-decl", true);
     return docFactory.newDocumentBuilder();
   }
 
+  @Bean
+  public TransformerFactory transformerFactory() {
+    var factory = TransformerFactory.newInstance();
+    factory.setAttribute(XMLConstants.ACCESS_EXTERNAL_DTD, "");
```

Comment:
This attribute is set in the handle manager:

`factory.setFeature(XMLConstants.FEATURE_SECURE_PROCESSING, true);`

Spring boot identified that as a security concern for me.

Reply:
Will add this